DSharpPlus/Entities/User/DiscordActivity.cs

```diff
@@ -297,6 +324,11 @@ public sealed class DiscordRichPresence
         /// </summary>
         public ulong? PartyId { get; internal set; }
 
+        /// <summary>
+        /// Gets the buttons.
+        /// </summary>
+        public IReadOnlyList<string> Buttons { get; internal set; }
```

Comment:
Why are buttons strings here? They should be [Button objects](https://discord.com/developers/docs/topics/gateway#activity-object-activity-buttons)

Reply:
On receiving, they are only an array of strings

Reply:
Ah, didn't read their description